### ISSUE TITLE
[BUG] Update via web interface migrate to dev branch

### ISSUE THREAD

**Issue by RemiCab** [FreshRSS/FreshRSS#4949]
**Describe the bug**
An update from a stable version via the web interface conduct to installing a dev version.

**To Reproduce**
1. Go to the update page
2. Click on apply update
4. See that the installation is now on a dev tag

**Expected behavior**
An update from a stable version should go to the next stable version (as it used to be)

**Screenshots**
![image](https://user-images.githubusercontent.com/8901633/208508342-5b6b84b3-7371-4b34-9c25-a9d594af56ce.png)

**Environment information (please complete the following information):**
- Device: Computer
- OS: Fedora 37
- Browser: Firefox 108
- FreshRSS version: 1.21.0-dev
- Database version: Postgresql 11
- PHP version: PHP 7.4
- Installation type: on premise via git

**Additional context**
I tired to reinstall FreshRSS from git following the documentation and was able to reproduce the issue. As I understand it seems to be the expected behavior since the migration to edge branch. Nevertheless if it is indeed the case, this is a change of behavior that was not explicated during the update, and is not reflected in the documentation. Moreover this left with no means than zip installation to stay on a stable version.

**Comment by stefangehn** [comment]
I wonder if this is caused by the fact that neither the 1.20.2 tag nor the commit tagged by it are part of the repository. A manual update by switching branches/tags isn't even possible via git at the moment.

This can also be seen when opening https://github.com/FreshRSS/FreshRSS/tree/1.20.2 or https://github.com/FreshRSS/FreshRSS/commit/27b71ffa99f7dff013fb8d51d020ed628e0d2ce6 since they both contain a warning at the top saying "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

**Comment by math-GH** [comment]
Is it reproduceable or do you understand what happend here? @Alkarex @marienfressinaud

**Comment by Frenzie** [comment]
Presumably that means the wrong commit was tagged by accident.

**Comment by Alkarex** [comment]
Our built-in Web update needs a rewrite https://github.com/FreshRSS/FreshRSS/pull/1760

The reason for the issue is that have been using a git tag `latest` to track the latest stable release https://github.com/FreshRSS/FreshRSS/releases/tag/latest but it would indeed be easier to make `latest` a branch instead of a tag so that the code of our Web updater would stick to the same release type.

Any objection @Frenzie for going back to a branch instead of a tag for tracking `latest`?

**Comment by Frenzie** [comment]
If I recall correctly it went against my intuition that `master` (that is, the default branch) was stable instead of development. I think that's still likely to be true for new contributors. Additionally it made it look like there hadn't been activity for months at first glance.

tl;dr If by `latest` you mean stable and `edge` remains the default then that's all good.

**Comment by Alkarex** [comment]
Yes, `edge` remains default

**Comment by Alkarex** [comment]
Done. Please check https://github.com/FreshRSS/FreshRSS/pull/5148
